.github/workflows/black-duck-security-scan-ci.yml

```
@@ -26,6 +26,7 @@ jobs:
       pull-requests: write
       security-events: write
       actions: read
+    if: ${{ vars.BLACKDUCKSCA_URL != '' && secrets.BLACKDUCKSCA_TOKEN != '' }}
```

actionlint (1.7.10)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)